style.css

```diff
@@ -484,7 +488,363 @@ footer {
+    width: 100%;
+  }
+
+  /* header/ navbar media query start */
+
+  .navbar-container {
+    min-height: 72px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .nav-ul {
+    width: 50%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .nav-ul li a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: var(--color-white);
+  }
+
+  /* header/ navbar media query end */
+
+  /* headline/homepage media query star */
+
+  .section {
+    min-height: 500px;
+    background-color: var(--color-black);
+    background-image: url("./images/desktop-1.svg");
+    background-repeat: no-repeat;
+    background-size: 50%;
+    background-position-x: 100%;
+    background-position-y: 90%;
+  }
+
+  .headline-container {
+    width: 50%;
+    flex-direction: row-reverse;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 85%;
+    height: 100%;
+    padding-right: 10%;
+  }
+
+  .headline-text h1 {
+    font-size: 50px;
+  }
+
+  .headline-text h3 {
+    font-size: 25px;
+  }
+
+  .social-media-icons {
+    width: 15%;
+    margin-top: 25%;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  /* work section media query for desktop version */
+
+  .works-container {
+    width: 85%;
+    min-height: 800px;
+    margin: 1px 10%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-title-dv {
+    width: 100%;
+    height: 100%;
+    margin-bottom: 0;
+    display: flex;
+    align-items: center;
+  }
+
+  .works-title {
+    font-size: 16px;
+    line-height: 20px;
+    margin: 2px;
+    margin-top: 55px;
+    width: 20%;
+  }
+
+  .work-title-dv hr {
+    margin: 0;
+    width: 80%;
+    height: 1px;
+    background-color: #0d151e;
+  }
+
+  .feature-area {
+    height: 50%;
+    display: flex;
+    align-items: flex-start;
+    margin-top: 0;
+    margin-bottom: 50px;
+  }
+
+  .feature-image {
+    width: 60%;
+    height: 70%;
+    margin: 0 10px 0 0;
+  }
+
+  .post-text {
+    display: flex;
+    flex-direction: column;
+    margin: 0 0;
+    width: 40%;
+  }
+
+  .title-post {
+    font-size: 20px;
+    margin-top: 0;
+  }
+
+  .post-description {
+    font-size: 80%;
+    line-height: 95%;
+    height: 70%;
+  }
+
+  .tag-ul-1 {
+    display: flex;
+    justify-content: space-around;
+    width: 70%;
+    height: 10%;
+  }
+
+  .tag-ul-1 li {
+    font-size: 8px;
+    line-height: 4px;
+    padding: 12px 8px;
+  }
+
+  form button {
+    margin-left: 10%;
+  }
+
+  .post-text button {
+    height: 35px;
+    width: 40%;
+    font-size: 95%;
+  }
+
+  .work-cards {
+    display: none;
+  }
+
+  .desktop-cards {
+    display: grid;
+    grid-template: repeat(2, minmax(350px, 1fr)) / repeat(3, minmax(220px, 1fr));
+    grid-gap: 2%;
+  }
```

Comment:
- [ ] As required by the last code reviewer, please 🙏🏽 adjust the space between the cards and also between the buttons of the skills and add the hover effect at each card to be similar to the Figma design. 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/214889907-95d3f882-c616-4bd3-bb42-009af32168db.png) | ![image](https://user-images.githubusercontent.com/58642949/214889190-093e3b3d-22c5-4cbb-a452-ace263d97e4d.png) |

Reply:
@franclobo each card's hover effects aren't a requirement